OOPAssignment1/Analyse.cs

```diff
@@ -8,29 +8,49 @@ namespace CMP1903M_Assessment_1_Base_Code
 {
     public class Analyse
     {
-        //Handles the analysis of text
-
-        //Method: analyseText
-        //Arguments: string
-        //Returns: list of integers
-        //Calculates and returns an analysis of the text
-        public List<int> analyseText(string input)
+        /// <summary>
+        /// Calculates and returns an analysis of the text
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns>List of integers corresponding to text analysis</returns>
+        public List<int> AnalyseText(string input)
         {
-            //List of integers to hold the first five measurements:
-            //1. Number of sentences
-            //2. Number of vowels
-            //3. Number of consonants
-            //4. Number of upper case letters
-            //5. Number of lower case letters
             List<int> values = new List<int>();
-            //Initialise all the values in the list to '0'
-            for(int i = 0; i<5; i++)
+
+            //List initialisation
+            for (int i = 0; i < 5; i++)
             {
                 values.Add(0);
             }
 
+            char[] endOfSentance = { '.', '!', '?' }; //Characters which indicate the end of a sentance. Ellipsis will create problems.
+
```

Comment:
Misspelled Sentence, consistency in mispelling however, so it doesn't affect the program when it runs.

Reply:
Good spot, thanks